### ISSUE TITLE
What you think to add touch control?

### ISSUE THREAD

**Issue by JTrotta** [LITTOMA/RadialGauge#1]

**Comment by LITTOMA** [comment]
Yes, I am planning to add interactive to the gauge control. Just like [the radial gauge in windows community toolkit](https://learn.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.controls.radialgauge?view=win-comm-toolkit-dotnet-7.1).
